### ISSUE TITLE
overfitting when train sdmgr model on custom dataset

### ISSUE THREAD

**Issue by mrlihellohorld** [open-mmlab/mmocr#607]
Thank you for your awesome works. I used my custom dataset training sdmgr model, the loss of training  is low, but the F1 of testing  is also very low. Mabey there existed the phenomenon of overfitting. so, I have some questions, hope to get your help.
1. In order to solve the overfitting problem, Is there a way of Data Arguementation when training sdmge model? 
2. I use this model to classify the news pictures in the subtitles,such 标题，人名介绍，对白字幕etc. Now, extracted text feature and image feature, can we extract  temporal feature(like i3d) as input? 
3. Now, the F1 of Caption  is very low, can we add weight when calculate the loss? Could you tell me how to do this specifily
My training log is as follows.Sample data sets and the distribution is as follows:
[log.log](https://github.com/open-mmlab/mmocr/files/7578253/log.log)
Sample data of test dataset as follows: 
[test.txt](https://github.com/open-mmlab/mmocr/files/7578256/test.txt)
Note: There are a lot of samples in 'other' category,so F1 of other is very high. There are none sample in category 5 in testdate, so the f1 is 0.

**Comment by gaotongxiao** [comment]
BTW, if you intended to link `主持人` and `马迟` as a pair, KIEOpenSet suits better for you. Please read through our [tutorial](https://mmocr.readthedocs.io/en/latest/tutorials/kie_closeset_openset.html) and figure out the better option.

**Comment by rajashreem52** [comment]
Can you ple

> Thank you for your awesome works. I used my custom dataset training sdmgr model, the loss of training is low, but the F1 of testing is also very low. Mabey there existed the phenomenon of overfitting. so, I have some questions, hope to get your help.
> 
> 1. In order to solve the overfitting problem, Is there a way of Data Arguementation when training sdmge model?
> 2. I use this model to classify the news pictures in the subtitles,such 标题，人名介绍，对白字幕etc. Now, extracted text feature and image feature, can we extract  temporal feature(like i3d) as input?
> 3. Now, the F1 of Caption  is very low, can we add weight when calculate the loss? Could you tell me how to do this specifily
>    My training log is as follows.Sample data sets and the distribution is as follows:
>    [log.log](https://github.com/open-mmlab/mmocr/files/7578253/log.log)
>    Sample data of test dataset as follows:
>    [test.txt](https://github.com/open-mmlab/mmocr/files/7578256/test.txt)
>    Note: There are a lot of samples in 'other' category,so F1 of other is very high. There are none sample in category 5 in testdate, so the f1 is 0.

Can you please tell me how did you format your custom dataset to mmocr format? Did you use dataset prepareparer? Did you create a custom dataset type or used wildrecipt as dataset type?
